middleware/lib/claim-client.js

```diff
@@ -77,6 +135,16 @@ class ClaimClient {
     return this._decode(data);
   }
 
+  /**
+   * Retrieve a single file for a given claim.
+   *
+   * @param {string} vin - The VIN of the claim.
+   * @param {Protos.File} fileProto - The file proto object.
+   */
+  async getFile(vin, fileProto) {
```

Comment:
**Note:** See my notes in the `getFile` function in the `s3.js` file below for important details about this.